src/app/pages/account/users/store/user.effects.ts

```diff
@@ -41,23 +41,7 @@ export class UserEffects {
     }),
   ));
 
-  // TODO: Two types of subscription need to be refactored into one in WebSocketService.
-  subscribeToUpdates$ = createEffect(() => this.actions$.pipe(
-    ofType(usersLoaded),
-    switchMap(() => {
-      return this.ws.subscribe('user.query').pipe(
-        filter((event) => !(event.msg === ApiEventMessage.Changed && event.cleared)),
-        map((event) => {
-          switch (event.msg) {
-            case ApiEventMessage.Added:
-              return userAdded({ user: event.fields });
-            case ApiEventMessage.Changed:
-              return userChanged({ user: event.fields });
-          }
-        }),
-      );
-    }),
-  ));
+  // userAdded() and userChanged() are dispatched from the User Form
```

Comment:
I think a slight benefit to the old system was that if for whatever reason new user is added in new tab / by another user (if even possible), this would have updated it.

Reply:
I completely agree. The reason is `ws.subscribe()/ws.sub()` for `user.create|user.update|user.query` endpoints does not produce websocket messages on adding/editing user.
Websocket output right after form submitted:
![image](https://user-images.githubusercontent.com/351613/159860575-54b155da-9207-4ef6-ae0f-97d345c46566.png)
Based on that, I getting `id` in response of `user.query`, then making a call `user.query` and dispatch `userAdded()` or `userChanged()`.